### DISCUSSION TITLE
Multiple RS485 Interfaces on one Raspberry Pi

### DISCUSSION THREAD

**Discussion by marthoc** [tagyoureit/nodejs-poolController#1001] [RS485]
This is more of a practical question, but I am currently planning out a build for njsPC. I have a Hayward MaxFlo VS500 pump that through trial and error I have determined communicates at 19,200 baud. I also have a Hayward Aquarite LS that communicates at 9,600 baud, so I cannot have the two on the same RS485 interface. What's the easiest way to have multiple interfaces? Can I use a Sequent card for one interface and then another Pi Hat RS485 interface for another?

**Comment by rstrouse** [comment]
Your easiest route is to use a USB dongle and the Sequent interface.  Many of the hats out there hook the same UART and will not work.

**Comment by TopJim** [comment]
I did mine using a passive USB hub and three dongles. I have three RS-485 busses on mine: 1 for IC20 on spa, 1 for IC40 on pool, and 1 for intellicenter.
https://www.amazon.com/dp/B00L2442H0?psc=1&ref=ppx_yo2ov_dt_b_product_details
https://www.amazon.com/dp/B07SD65BVF?psc=1&ref=ppx_yo2ov_dt_b_product_details
![IMG_2432](https://github.com/user-attachments/assets/5e7cadcd-9ac6-4668-8c58-cf6872bb970b)
